packages/core/src/lib/translate/translate.ts

```diff
@@ -54,6 +56,9 @@ const findObjectPropByStringPath = (obj: any, prop: string): unknown => {
 export const translate = (path: string) => {
   let lang = window.navigator.languages ? window.navigator.languages[0] : null;
   lang = lang || window.navigator.language;
+  if (devLanguage !== null) {
```

Comment:
Based on sugestion ` useAllowOnlyLanguage && (lang = allowOnlyLanguage);` can be used instead of `if{...} block

Reply:
My opinion is that `if{...}` block already does its function. For this method, you will need to add a new variable `useAllowOnlyLanguage=true`  and `allowOnlyLanguage`. So instead of change language only in one place `allowOnlyLanguage="es"`, developers would need to add also `useAllowOnlyLanguage=true`